### PR TITLE
Blog comments simple

### PR DESCRIPTION
**Pull request by erh**

### CODE REVIEW

File: blog-comments-simple/Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+readme:
```
Comment (edaniels) [inline blog-comments-simple/Makefile]: We should probably remove this, no? Considering it's going into a repo where we will just maintain the README.

File: blog-comments-simple/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+Next we'll startup a simple python webserver
+
+     python -m SimpleHTTPServer
+
+Now browse to [the page](http://localhost:8000/blog.html)
+
+Ok, we have our basic page, time to add comments.
+
+Browse to your stitch application home page, on it you should see a script tag to load the stitch sdk and connect.
+Copy the first few lines that should look something like:
```
Comment (edaniels) [inline blog-comments-simple/README.md]: The code excerpt should be changed according to comments in good.html.

File: blog-comments-simple/README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+               const client = new stitch.StitchClient('eliot1-qffsf');
+               const db = client.service('mongodb', 'mongodb1').db('blog');
+
+For the `db` argument, change the name to `blog` from whatever was there before.
+
+This is the basic setup code.
+
+Next we are going to add an onLoad handler.
+
+So add a function in the script block:
```
Comment (edaniels) [inline blog-comments-simple/README.md]: Delete So. add -> Add

File: blog-comments-simple/README.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+         function displayCommentsOnLoad() {
+             client.anonymousAuth().then(displayComments)
+         }
+
+Add make your body tag look like:
```
Comment (edaniels) [inline blog-comments-simple/README.md]: Add -> And?

File: blog-comments-simple/README.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+    <body onLoad="displayCommentsOnLoad()">
+
+That function firsts logs the user into stitch anonymously, and then dislpays any comments in the database.
```
Comment (edaniels) [inline blog-comments-simple/README.md]: firsts -> first

File: blog-comments-simple/index.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
+<h2>Adding comments to a blob with Stitch</h2>
```
Comment (edaniels) [inline blog-comments-simple/index.html]: Delete this file

File: blog-comments-simple/good.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
+<html>
+    <head>
+        <script src="https://s3.amazonaws.com/stitch-sdks/js/library/298a2b586d91d462099e5d9f66fba0a687837abe/stitch.min.js"></script>
```
Comment (edaniels) [inline blog-comments-simple/good.html]: <script src="https://s3.amazonaws.com/stitch-sdks/js/library/stable/stitch.min.js"></script>

File: blog-comments-simple/good.html [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+        <script>
+         const client = new stitch.StitchClient('eliot1-qffsf');
```
Comment (edaniels) [inline blog-comments-simple/good.html]: Might want to put a placeholder value here

File: blog-comments-simple/good.html [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+         const db = client.service('mongodb', 'mongodb1').db('blog');
```
Comment (edaniels) [inline blog-comments-simple/good.html]: The service type will always be `mongodb-atlas` and the name `mongodb-atlas`

File: blog-comments-simple/good.html [hunk 10]
```diff
@@ -0,0 +1,38 @@
+<html>
```
Comment (edaniels) [inline blog-comments-simple/good.html]: Should we get rid of this file?

File: blog-comments-simple/pieces/middle1.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
+
```
Comment (edaniels) [inline blog-comments-simple/pieces/middle1.md]: Delete this file

File: blog-comments-simple/pieces/opening.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
+Adding comments to a blog with Stitch
```
Comment (edaniels) [inline blog-comments-simple/pieces/opening.md]: Delete this file

File: blog-comments-simple/README.md [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+You should now be able to add comments to your blog!
+
+The entire thing looks like:
+
+
+
+
+
+
+     <html>
```
Comment (edaniels) [inline blog-comments-simple/README.md]: See comments in the good.html file

File: .gitignore [hunk 14]
```diff
@@ -3,4 +3,5 @@ node_modules
 npm-debug.log
```
Comment (edaniels) [inline .gitignore]: Can remove this too

File: blog-comments-simple/Makefile (hunk 1 above)
Comment (erh) [inline blog-comments-simple/Makefile]: I don't want to remove the other files.
I think they are useful for someone playing around.

File: blog-comments-simple/README.md (hunk 3 above)
Comment (erh) [inline blog-comments-simple/README.md]: done

File: blog-comments-simple/README.md (hunk 4 above)
Comment (erh) [inline blog-comments-simple/README.md]: done

File: blog-comments-simple/README.md (hunk 5 above)
Comment (erh) [inline blog-comments-simple/README.md]: done

File: blog-comments-simple/index.html (hunk 6 above)
Comment (erh) [inline blog-comments-simple/index.html]: yeah

File: .gitignore (hunk 14 above)
Comment (erh) [inline .gitignore]: done

File: blog-comments-simple/pieces/opening.md (hunk 12 above)
Comment (erh) [inline blog-comments-simple/pieces/opening.md]: no

File: blog-comments-simple/pieces/middle1.md (hunk 11 above)
Comment (erh) [inline blog-comments-simple/pieces/middle1.md]: no

File: blog-comments-simple/good.html (hunk 9 above)
Comment (erh) [inline blog-comments-simple/good.html]: done

**Pull request closed without merging**